lib/internal/Magento/Framework/Mail/Template/AttachmentTransportBuilder.php

```diff
@@ -0,0 +1,219 @@
+<?php
+/**
+ * Copyright © 2013-2017 Magento, Inc. All rights reserved.
```

Comment:
Please correct the copyright. Tests failed with error:
`Copyright is missing or has wrong format`